centrosome/neighmovetrack.py

```diff
@@ -1,13 +1,19 @@
+from __future__ import absolute_import
 import math
 import copy
 import numpy
 import scipy.ndimage.measurements
-import centrosome.lapjv
+from . import lapjv
+import six
+from six.moves import range
+from six.moves import zip
 
 invalid_match = 1000000  # limiting the choices of the algorithms
 
 
-def euclidean_dist((x1, y1), (x2, y2)):
+def euclidean_dist(xxx_todo_changeme, xxx_todo_changeme1):
```

Comment:
I assume this still needs to be changed?

Reply:
HOLY MOLY!

I had no idea that:
- modernize could make changes that actually break the code
- `def func((a, b), c)` was valid Python
- that this was changed in Python 3

My general approach to Python 3 compatibility is:
- run `python-modernize -w -n .` on a codebase
- fix all the failing tests

This means that many redundant `list()` wrappers will remain and that anything broken that is not covered by a test will remain broken. That's what happened here and in many of the redundant wrappers. (Sometimes I remove these but only when they are completely obvious and I run into them during my test repairs.)

I'll update this *and* add a test. ;)

Reply:
Oh I just saw that actually the fix was not breaking, it did the right thing, it just didn't name the variables nicely. Whew! That's a load off my mind. =) I might suggest that concatenating the variable names in the tuple would be a better strategy. =)